RunCat.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30320.27
+# Visual Studio Version 17
```

Comment:
Is the diff of this file necessary to achieve your goal?

Reply:
I do not think so, but did not test this thought either.